systemtests/src/test/java/io/enmasse/systemtest/common/api/ApiServerTest.java

```diff
@@ -247,4 +248,14 @@ void testCreateAddressResource() throws Exception {
         assertThat(addresses.size(), is(3));
         TestUtils.waitForDestinationsReady(addressApiClient, addrSpace, new TimeoutBudget(5, TimeUnit.MINUTES), anycast, multicast, longname);
     }
+
+    @Test
```

Comment:
You don't need this test, bec ause you already test deleteAllAddressSpaces method in another test.

Reply:
test deleted